pkg/k8shandler/elasticsearch.go

```diff
@@ -322,7 +351,44 @@ func GetMinMasterNodes(clusterName, namespace string, client client.Client) (int
 	return masterCount, payload.Error
 }
 
-func GetClusterHealth(clusterName, namespace string, client client.Client) (string, error) {
+func GetClusterHealth(clusterName, namespace string, client client.Client) (api.ClusterHealth, error) {
+
+	clusterHealth := api.ClusterHealth{}
+
+	payload := &esCurlStruct{
+		Method: http.MethodGet,
+		URI:    "_cluster/health",
+	}
+
+	curlESService(clusterName, namespace, payload, client)
```

Comment:
should we short circuit here if `payload.Error` indicates an error?

Reply:
I wasn't planning on doing that here but we could to save trying to crawl an empty map... let me add that